templates/helm/templates/irsa.yaml.tpl

```diff
@@ -0,0 +1,30 @@
+{{- if and .Values.aws.accountId .Values.aws.oidcProvider }}
+apiVersion: iam.services.k8s.aws/v1alpha1
+kind: Role
+metadata:
+  name: {{ .Values.serviceAccount.name }}
```

Comment:
We might want to make the role name a separate property from the service account, otherwise installing ACK controllers in 2 clusters within the same account would have conflicting roles.